datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -9618,6 +9623,11 @@ type Privileges {
   Granted Privileges
   """
   privileges: [String!]!
+
+  """
+  Details about how each policy was evaluated
+  """
+  evaluationDetails: [PolicyEvaluationDetail]
```

Comment:
It would be a nice-to-have if the evaluationDetails had a single `pass` / `fail` which is an OR of all the policies. Thinking of a case where a DataHub instance has hundreds if not thousands of policies.

Reply:
We do know what is the final privileges granted already. This is for having granular debugging info.